tests/test_meshtastic_utils.py

```
@@ -909,27 +1114,64 @@ def test_connect_meshtastic_retry_exhausted(
-@patch("mmrelay.meshtastic_utils.asyncio.sleep")
+@patch("mmrelay.meshtastic_utils.asyncio.sleep", new_callable=AsyncMock)
 @patch("mmrelay.meshtastic_utils.logger")
-async def test_reconnect_attempts_connection(
-    mock_logger, mock_sleep, mock_connect, reset_meshtastic_globals
+def test_reconnect_attempts_connection(
+    _mock_logger,
+    mock_sleep,
+    mock_connect,
+    mock_get_loop,
+    _mock_is_service,
+    reset_meshtastic_globals,
```

Ruff (0.14.6)

Unused function argument: `reset_meshtastic_globals`

(ARG001)